jerry-core/api/jerry.c

```diff
@@ -4332,11 +4332,15 @@ jerry_get_typedarray_buffer (jerry_value_t value, /**< TypedArray to get the arr
 } /* jerry_get_typedarray_buffer */
 
 /**
- * Create an object from JSON
+ * Parse the given JSON string to create a jerry_value_ŧ.
```

Comment:
That suffix looks a bit odd, it it a regular `t` character?

Reply:
Should be fixed now